### PR TITLE
Adding loading pattern to profiles page + hide 0 until after page loads

### PR DESCRIPTION
**Pull request by vinay033**
Signed-off-by: vinay033 <vsharma@chef.io>

### :nut_and_bolt: Description
As an Automate User, I would like the following UI pages not to show a value of 0 for counts while data from the backend in pulled in order to render them. It creates a situation where the page is visited and for a few seconds, the user will see 0 counts and then as data is rendered, the counts change.
### :+1: Definition of Done
new changes video 
https://drive.google.com/file/d/131ST9sOB8xBg4LHQy1yMjg-g5pMrc2jg/view?usp=sharing
### :chains: Related Resources
fixes https://github.com/chef/automate/issues/311

### CODE REVIEW

**Comment by susanev** [comment]
@vinay033 could you make it so the overlay is just for the profiles page content and not the left-nav? otherwise, it looks good!

**Comment by susanev** [comment]
@vinay033 ignore my previous comment, we can come back and fix that in a separate task.

File: components/automate-ui/src/app/pages/+compliance/+scanner/containers/scanner/scanner.component.ts [hunk 1]
```diff
@@ -13,12 +13,22 @@ export class ScannerComponent implements OnInit {
   jobsCount$: Observable<number>;
   nodesCount$: Observable<number>;
 
+  // show spinner befor loading data
```
Comment (msorens) [inline components/automate-ui/src/app/pages/+compliance/+scanner/containers/scanner/scanner.component.ts]: s/befor/before/

File: components/automate-ui/src/app/pages/+compliance/+profile/+profile-overview/profile-overview.component.ts [hunk 2]
```diff
@@ -60,6 +60,11 @@ export class ProfileOverviewComponent implements OnInit, OnDestroy {
   // notification data
   downloadErrorVisible = false;
 
+  // show spinner befor loading data
```
Comment (msorens) [inline components/automate-ui/src/app/pages/+compliance/+profile/+profile-overview/profile-overview.component.ts]: s/befor/before/

Comment (vinay033) [reply components/automate-ui/src/app/pages/+compliance/+profile/+profile-overview/profile-overview.component.ts]: changes added :)

**Pull request merged**